### PR TITLE
PooledDataSource

### PR DESCRIPTION
**Pull request by DanielChanger**

### CODE REVIEW

File: src/main/java/com/example/App.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            }
+        }
+        System.out.println((System.nanoTime() - start) / 1000_000 + " ms");
+        System.out.println(total);
+
+
+    }
+
+    private static DataSource initializePooledDataSource() {
+        int threadCount = 10;
```
Comment (tboychuk) [inline src/main/java/com/example/App.java:31]: It's a connection count, not a thread count. 😉

**Review by tboychuk — APPROVED** [review]
@DanielChanger looks clear! 👍

File: src/main/java/com/example/App.java (hunk 1 above)
Comment (DanielChanger) [reply src/main/java/com/example/App.java:31]: good catch :)

Comment (DanielChanger) [reply src/main/java/com/example/App.java:31]: Also, not datasourcePool but connectionPool))) Seems like I absolutely didn't care of naming, haha

**Pull request merged**